Cargo.toml

```diff
@@ -6,6 +6,11 @@ members = [
     "hint_accountant",
     "examples/wasm-demo",
 ]
+default-members = [
```

Comment:
Just to check @fmoletta 
Why are we adding this default-members?

Reply:
The problem we want to fix is that the build fails due to the wasm-demo needing the compiled cairo files in order to build.
Someone that wants to build the cairo-vm crate shouldn't have to compile a cairo file in order to build a demo for the vm crate to build.
Therefore, by setting the default members we can build the core crates (vm, cli & felt) with cargo/make build without building the wasm demo and running into compilation errors